From 888330527f94df225b9952b9b11256b510932565 Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Tue, 21 Oct 2014 15:46:15 -0500
Subject: [PATCH] Increase attribute upper bounds by a factor of 20 (except for
 cpus_per_node, which has already gotten bumped recently) (SOFTWARE-1638)

---
 osg_configure/configure_modules/gip.py | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/osg_configure/configure_modules/gip.py b/osg_configure/configure_modules/gip.py
index cf02c9b..fbe8ccd 100644
--- a/osg_configure/configure_modules/gip.py
+++ b/osg_configure/configure_modules/gip.py
@@ -78,13 +78,13 @@ SE_BANNED_ENTRIES = {
    "srm_endpoint": "httpg://srm.example.com:8443/srm/v2/server",
 }
 
-SC_ATTRIBUTE_RANGES = {'SF00': (500, 5000),
-                       'SI00': (500, 5000),
-                       'HEPSPEC': (2, 50),
-                       'ram_mb': (500, 102400),
-                       'swap_mb': (500, 102400),
-                       'cpus_per_node': (1, 128),
-                       'cores_per_node': (1, 256)}
+SC_ATTRIBUTE_RANGES = {'SF00': (500, 100000),
+                       'SI00': (500, 100000),
+                       'HEPSPEC': (2, 1000),
+                       'ram_mb': (500, 2048000),
+                       'swap_mb': (500, 2048000),
+                       'cpus_per_node': (1, 640),
+                       'cores_per_node': (1, 2560)}
 
 # Error messages
 mount_point_error = """\
-- 
2.0.1

